src/formatter.rs

```diff
@@ -430,7 +450,8 @@ mod tests {
             r#"
         colors.fg = { fg = "red" }
         colors.bg = { bg = "blue" }
```

Comment:
Same here, can we use a different label than 'bg'? And in the future commit, 'bold'. :)